File: cs50/sentimental-cash/cash.py
from cs50 import get_float

while True:
    try:
        n = get_float("Change: ")
        if (n >= 0.01):
            break
    except:
        pass

m = 0
n = n * 100

while True:
    if (n >= 25):
        n -= 25
        m += 1
    elif (n >= 10):
        n -= 10
        m += 1
    elif (n >= 5):
        n -= 5
        m += 1
    elif (n >= 1):
        n -= 1
        m += 1
    if (n == 0):
        break

print(f"{m}")

File: cs50/cash/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Change owed: ");
    }
    while (n < 1);

    int m = 0;
    do
    {
        if (n >= 25)
        {
            n = n - 25;
            m = m + 1;
        }
        else if (n >= 10)
        {
            n = n - 10;
            m = m + 1;
        }
        else if (n >= 5)
        {
            n = n - 5;
            m = m + 1;
        }
        else if (n >= 1)
        {
            n = n - 1;
            m = m + 1;
        }
    }
    while (n != 0);

    printf("%i\n", m);
}
